tests/reporter/kubernetes.go

```diff
@@ -173,6 +181,133 @@ func (r *KubernetesReporter) logVMIs(virtCli kubecli.KubevirtClient) {
 	fmt.Fprintln(f, string(j))
 }
 
+func (r *KubernetesReporter) logDMESG(virtCli kubecli.KubevirtClient, since time.Time) {
+
+	logsdir := filepath.Join(r.artifactsDir, "nodes")
+
+	if err := os.MkdirAll(logsdir, 0777); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create directory: %v\n", err)
```

Comment:
Can you say what directory in the message? Not sure the error will always contain it.

Reply:
Done.